### PR TITLE
Quickstart rewrite

### PR DESCRIPTION
**Pull request by chrisurwin**
Rewrite of the Quick Start section of docs

### CODE REVIEW

File: content/rancher/v2.x/en/quick-start-guide/deployment/digital-ocean-qs/_index.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+- [Terraform](https://www.terraform.io/downloads.html) - This is used to provision the server and cluster to Digital Ocean
+
+
+## Getting going
+
+1. Clone [Rancher Quickstart](https://github.com/rancher/quickstart) to a folder using `git clone https://github.com/rancher/quickstart`
+
+2. Go into the folder containing the terraform file by executing `cd quickstart/do`
+
+3. Move the file `terraform.tfvars.example` to `terraform.tfvars` and edit (see inline explanation, the minimum you will need to do is add your Digital Ocean Access Key)
```
Comment (MBishop17) [inline content/rancher/v2.x/en/quick-start-guide/deployment/digital-ocean-qs/_index.md]: 1. I don't follow what you mean here by "move." Do you mean "rename"?

2. I think there's a word missing after "edit". 

3. "(see inline explanation, the minimum you will need to do is add your Digital Ocean Access Key)". Parenthesis aren't needed here. And replace the comma with a period. 

4. We need to tell people how to get the key.

File: content/rancher/v2.x/en/quick-start-guide/deployment/digital-ocean-qs/_index.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    >**Wait for the the following**
+	> ```
+	Apply complete! Resources: 4 added, 0 changed, 0 destroyed. 
+	  Outputs: 
+	  rancher-url = [ 
+              https://xxx.xxx.xxx.xxx 
+      ]
+	```
+
+6. Paste the URL into the browser and log in (default password is `admin`)
```
Comment (MBishop17) [inline content/rancher/v2.x/en/quick-start-guide/deployment/digital-ocean-qs/_index.md]: we should replace `URL` with `rancher-url` just to be clear.

File: content/rancher/v2.x/en/quick-start-guide/deployment/quickstart-vagrant/_index.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+1. Clone [Rancher Quickstart](https://github.com/rancher/quickstart) to a folder using `git clone https://github.com/rancher/quickstart`
+
+2. Go into the folder containing the terraform file by executing `cd quickstart/vagrant`
+
+3. Edit the config.yaml if required (you can change the number of nodes and the memory allocations if required)
+
+4. To initiate the creation of the environment run `vagrant up`
+
+5. Once provisioning has finished go to `https://172.22.101.100` in the browser (default user/password is `admin/password1` you can change this in the config.yaml prior to running the `vagrant up` command)
```
Comment (MBishop17) [inline content/rancher/v2.x/en/quick-start-guide/deployment/quickstart-vagrant/_index.md]: Break the parenthesis into multiple sentences.

File: content/rancher/v2.x/en/quick-start-guide/workload/quickstart-deploy-workload-ingress/_index.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+3.  Open the **Default** project.
+
+4.  Click on the **Load Balancing** tab.
+
+5.  Click **Add Ingress**.
+
+6.  Enter a name i.e. **hello**
+
+7.  In the **Target* field, drop down the list and choose the name that you set for your service.
```
Comment (MBishop17) [inline content/rancher/v2.x/en/quick-start-guide/workload/quickstart-deploy-workload-ingress/_index.md]: Close the strong font with another asterisk.

**Comment by JasonvanBrackel** [comment]
@MBishop17 can you review @chrisurwin updates based on your feedback?

**Pull request merged**